web/themes/custom/jcc_base/preprocess/view/views_view.preprocess.inc

```diff
@@ -59,3 +59,11 @@ function jcc_base_preprocess_views_view_unformatted__faq__block(&$variables) {
     ];
   }
 }
+
+/**
+ * Implements hook_preprocess_HOOK().
+ */
+function jcc_base_preprocess_views_view_summary_unformatted__glossary_terms(&$variables) {
```

Comment:
@baptisten you can move this whole function into its own file:
preprocess/view/views-view-summary-unformatted--glossary-terms.preprocess.inc

Reply:
Right. I just committed the change @ivanbueno-jcc